docs/cookbook/headers.md

```diff
@@ -0,0 +1,35 @@
+# Headers and footers
+It is quite common to use a unified header and/or footer for all slides of a slide deck. You can
+achieve this using *Elsie* e.g. by creating a factory function for each slide:
+```elsie,type=lib
+def init_slide(slide, header_text: str):
+    # Create a header with some background color
+    header = slide.box(width="fill", height="10%").rect(bg_color="#23363A")
+
+    # Put some text into the header
+    row = header.box(horizontal=True, padding=10)
```

Comment:
Why the row box has horizontal=True?

Maybe we can show here right aligned it is "more" complicated (i.e. x = "[100%]" + some right padding)